src/detection/detection-wrapper.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyles } from 'detection/style'
+
+export const DetectionWrapper = ({ children }) => <Grid container justify={'center'} alignContent={'center'}
+                                                        className={useStyles().root}>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125618861-93138b53-b821-495f-a143-6ca0b2ddf42d.png)
This one's on me. If all your styles are inside the `root` field, you can provide them to your material ui components by using the `classes` prop.

Reply:
Let's change it to this way in all files touched in this pr.

Reply:
done